packages/message-parser/benchmarks/parser.bench.ts

```
@@ -166,6 +174,64 @@ const categories: BenchCategory[] = [
+
+	for (let i = 0; i < args.length; i++) {
+		const flag = args[i];
+		const value = args[i + 1];
+
+		if ((flag === '--skip' || flag === '--only') && value && !value.startsWith('--')) {
+			const names = value
+				.split(',')
+				.map((s) => s.trim().toLowerCase())
+				.filter(Boolean);
+
+			names.forEach((n) => result[flag === '--skip' ? 'skip' : 'only'].add(n));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)